sklearn/preprocessing/_encoders.py

```diff
@@ -239,13 +261,51 @@ class OneHotEncoder(_BaseEncoder):
     dtype : number type, default=float
         Desired dtype of output.
 
-    handle_unknown : {'error', 'ignore'}, default='error'
-        Whether to raise an error or ignore if an unknown categorical feature
-        is present during transform (default is to raise). When this parameter
-        is set to 'ignore' and an unknown category is encountered during
-        transform, the resulting one-hot encoded columns for this feature
-        will be all zeros. In the inverse transform, an unknown category
-        will be denoted as None.
+    handle_unknown : {'error', 'ignore', 'infrequent_if_exist'}, \
+                     default='error'
+        Specifies a methodology for handling unknown categories during
+        :meth:`transform`.
+
+        - 'error' : Raise an error if an unknown categorical feature
+          is present during transform.
+        - 'ignore' : When an unknown category is encountered during
+          transform, the resulting one-hot encoded columns for this feature
+          will be all zeros. In the inverse transform, an unknown category
+          will be denoted as None.
+        - 'infrequent_if_exist' : When an unknown category is encountered
```

Comment:
We might want to mention that the infrequent column will be added at the last position when enconding